CollisionOperator/CollisionProcessors/neutronCEimp_class.f90

```diff
@@ -168,6 +179,13 @@ subroutine init(self, dict)
          'Must generate fission sites implicitly when using implicit absorption')
     end if
 
+    ! Sets up the uniform fission sites field
+    if (self % uniFissSites) then
+      name = 'UFS'
```

Comment:
Similarly to weight windows, perhaps we should make the common name a constant in `universalVariables`?